### ISSUE TITLE
Crash when using --save parameter

### ISSUE THREAD

**Issue by yossizahn** [coderaiser/cloudcmd#247]
* **Version**: v12.6.1
* **Node Version**: v10.16.0
* **OS**: `Linux instance-1 4.9.0-9-amd64 #1 SMP Debian 4.9.168-1+deb9u2 (2019-05-13) x86_64 GNU/Linux`
* **Browser name/version**: n/a
* **Used Command Line Parameters**: --save
* **Changed Config**: no

```
$ cloudcmd --save
/usr/lib/node_modules/cloudcmd/bin/cloudcmd.js:223
    const caller = (fn) => fn();
                           ^

TypeError: fn is not a function
    at caller (/usr/lib/node_modules/cloudcmd/bin/cloudcmd.js:223:28)
    at main (/usr/lib/node_modules/cloudcmd/bin/cloudcmd.js:226:27)
    at Object.<anonymous> (/usr/lib/node_modules/cloudcmd/bin/cloudcmd.js:152:5)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
```

**Comment by yossizahn** [comment]
**Note:** This isn't due to using no other parameters in conjunction with `--save`, since this occurs when using other parameter also.

**Comment by coderaiser** [comment]
Thank you 🙂, landed in [v12.6.2](https://github.com/coderaiser/cloudcmd/releases/tag/v12.6.2) 🎉.
Is it helps?

**Comment by yossizahn** [comment]
Great! :+1:
